# EXCEPTION CASE 7
try:
    pass
    print("try scope")
    pass
except:
    print("foovar")
else:
    print("no exception raised")
